app/src/main/java/net/aiscope/gdd_app/ui/capture/ZoomController.kt

```diff
@@ -14,6 +14,8 @@ class ZoomController(
     private val cameraView: View
 ) {
 
+    var onZoomChangedListener : OnZoomChangedListener? = null
```

Comment:
Why not move this to the constructor? I don't see it ever reassigned, so it could be a `val` too.

Reply:
done